packages/overlay/src/index.tsx

```diff
@@ -73,157 +31,118 @@ export async function sendEvent(contentType: string, data: string | Uint8Array)
-
-  const fullPage = options.fullPage ?? isLoadedFromSidecar;
-  const showTriggerButton = options.showTriggerButton ?? !fullPage;
-  const injectImmediately = options.injectImmediately ?? (isLoadedFromSidecar || fullPage);
-  const debug = options.debug ?? document.location.hash.endsWith("debug");
+    debug = document.location.hash.endsWith("debug"),
+  } = initOptions;
 
   if (debug) {
     activateLogger();
   }
 
+  const sidecarBaseUrl = removeURLSuffix(sidecarUrl, "/stream");
   const finalExperiments = { ...DEFAULT_EXPERIMENTS, ...experiments };
 
-  // Sentry is enabled by default
-  const defaultIntegrations = () => [sentry({ injectIntoSDK: !isLoadedFromSidecar && !fullPage })];
+  const telemetry = initTelemetry({
+    injectIntoSDK: true,
+  });
 
-  const context: SpotlightContext = {
-    open: openSpotlight,
-    close: closeSpotlight,
-    experiments: finalExperiments,
-    sidecarUrl: sidecarBaseUrl,
-  };
-
-  const [initializedIntegrations] = await initIntegrations(integrations ?? defaultIntegrations(), context);
-
-  // build shadow dom container to contain styles
-  const docRoot = document.createElement("div");
-  docRoot.id = "sentry-spotlight-root";
-  const shadow = docRoot.attachShadow({ mode: "open" });
-  const appRoot = document.createElement("div");
-  if (fullPage) {
-    docRoot.style.height = "100%";
-    docRoot.style.backgroundColor = colors.indigo[950];
-    appRoot.style.height = "100%";
-  } else {
-    appRoot.style.position = "absolute";
-    appRoot.style.top = "0";
-    appRoot.style.left = "0";
-    appRoot.style.right = "0";
-  }
-  shadow.appendChild(appRoot);
-
-  const ssGlobal = createStyleSheet(globalStyles);
-  shadow.adoptedStyleSheets = [createStyleSheet(fontStyles), ssGlobal];
-
-  if (import.meta.hot) {
-    import.meta.hot.accept("./index.css?inline", newGlobalStyles => {
-      ssGlobal.replaceSync(newGlobalStyles?.default);
+  if (telemetry.setup) {
+    await telemetry.setup({
+      navigate: (path?: string) => {
+        trigger("navigate", path || "/");
+      },
+      sidecarUrl: sidecarBaseUrl,
     });
   }
 
-  const tabs = getPanelsFromIntegrations(initializedIntegrations, {});
+  const appRoot = document.createElement("div");
+  appRoot.id = "spotlight-root";
+  appRoot.style.height = "100vh";
+  appRoot.style.width = "100vw";
+
+  // Add styles
+  const styleElement = document.createElement("style");
+  styleElement.textContent = `${fontStyles}\n${globalStyles}`;
+  document.head.appendChild(styleElement);
 
-  const initialTab = startFrom || (tabs.length ? `/${tabs[0].id}` : "/no-tabs");
+  const initialTab = startFrom || "/traces";
   log("Starting from", initialTab);
 
-  // TODO: Shall we enable this for other cases too such as when we use it through Django SDK?
-  if (fullPage && (isLoadedFromSidecar || document.location.hash.startsWith("#spotlight"))) {
+  const isLoadedFromSidecar = new URL(sidecarUrl).origin === document.location.origin;
```

Comment:
This probably doesn't make any sense as "not loaded from sidecar" was the implication of being embedded. (yes technically you can load the UI from somewhere else but it was not the intention behind these checks -- this was mostly for instrumenting Spotlight itself without clobbering other people's sentry telemetry)